src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -556,11 +556,11 @@ namespace OpenLoco::Ui::WindowManager
         // Border flash invalidation
         for (Ui::Window* w = _windowsEnd - 1; w >= _windows; w--)
         {
-            if ((w->flags & WindowFlags::whiteBorderMask) != 0)
+            if (w->hasFlags(WindowFlags::whiteBorderMask))
             {
                 // TODO: Replace with countdown
-                w->flags -= WindowFlags::whiteBorderOne;
-                if ((w->flags & WindowFlags::whiteBorderMask) == 0)
+                w->flags &~ WindowFlags::whiteBorderOne;
```

Comment:
This looks wrong, you probably want:
```suggestion
                w->flags &= ~WindowFlags::whiteBorderOne;
```